src/channel.ts

```diff
@@ -93,6 +93,30 @@ export type PositionChannel = 'x' | 'y' | 'x2' | 'y2';
 
 export type GeoPositionChannel = 'longitude' | 'latitude' | 'longitude2' | 'latitude2';
 
+export function isGeoPositionChannel(c: Channel): c is GeoPositionChannel {
+  switch (c) {
```

Comment:
`contains(c, [LAT,LAT2,...])`?

Reply:
The current code is faster.

Reply:
I believe you but how do you know?

Reply:
switch is look up, contains is for loop?  It probably doesn't matter. Let's say I won't change because it doesn't matter.   But let me add tests.